kitodo-mediaserver-core/src/main/resources/config/default.yml

```diff
@@ -31,6 +41,11 @@ logging:
     file: "%d{yyyy-MM-dd HH:mm:ss} [%thread] %-5level %logger - %msg%n"
   path: /usr/local/kitodo/mediaserver/
 
+notifier:
+  emailFrom: "kitodo@example.org"
```

Comment:
In default.yml the email lists should be empty as you can not override it in local.yml.

Reply:
I guess this is okay here since it's not a list and we definitely need a sender address?

Reply:
Oh, yes, you are right. forget about this.